Rubberduck.VBEEditor/QualifiedModuleName.cs

```diff
@@ -55,11 +62,11 @@ public QualifiedModuleName(VBComponent component)
         /// Creates a QualifiedModuleName for a built-in declaration.
         /// Do not use this overload for user declarations.
         /// </summary>
-        public QualifiedModuleName(string projectName, string componentName)
+        public QualifiedModuleName(string projectName, string projectPath, string componentName)
         {
             _project = null;
-            _projectName = projectName;
-            _projectId = projectName.GetHashCode().ToString();
+            _projectName = projectName + ";" + projectPath;
```

Comment:
That constructor is only used for built-in declarations. Those aren't displayed in the toolbar right?